Tensile/TensileCreateLibrary.py

```diff
@@ -1330,7 +1330,7 @@ def splitExtraParameters(par):
   argParser.add_argument("RuntimeLanguage", help="Which runtime language?", choices=["OCL", "HIP", "HSA"])
   argParser.add_argument("--cxx-compiler",           dest="CxxCompiler",       choices=["hcc", "hipcc"],       action="store", default="hipcc")
   argParser.add_argument("--code-object-version",    dest="CodeObjectVersion", choices=["V2", "V3"], action="store", default="V3")
-  argParser.add_argument("--architecture",           dest="Architecture",      choices=["all", "gfx000", "gfx803", "gfx900", "gfx906", "gfx908"], action="store", default="all")
+  argParser.add_argument("--architecture",           dest="Architecture",      choices=["all", "gfx000", "gfx803", "gfx900", "gfx906:xnack-", "gfx908:xnack-"], action="store", default="all")
```

Comment:
I don't know enough of the plans but for backward compatibility should you have all gfx906, gfx906:xnack-, gfx906:xnack-  etc.  ?   Or in the mapping can you just strip off the ":flags" just like you did from the hip gcn arch? 
That may save you from further changes....

Reply:
At this point TargetID support is very inflexible.  As a result there is no backward compatibility.  @AlexBrownAMD will hopefully restructure the code to make it more flexible when he has some time.  Thanks.